nixos/modules/services/web-servers/nginx/default.nix

```
@@ -1226,6 +1226,7 @@ in
       ++ lib.optional cfg.recommendedZstdSettings pkgs.nginxModules.zstd;
 
     services.nginx.virtualHosts.localhost = mkIf cfg.statusPage {
+      serverAliases = [ "127.0.0.1" ];
```

```suggestion
      serverAliases = [ "127.0.0.1" "::1" ];
```
or is localhost resolving to that?

---

I don't have IPv6 enabled so not sure but I've added an alias for it as well.

---

> I don't have IPv6 enabled

That's still a thing?